gnovm/stdlibs/net/url/url_test.gno

```diff
@@ -0,0 +1,2222 @@
+// Copyright 2009 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package url
+
+import (
+	"bytes"
+	// encodingPkg "encoding"
```

Comment:
Was this intentionally left commented out?

Reply:
yes ! since it's copy/past from the orignal golang files, I prefere to simply comment instead of removing the line to easily find & fix if we add this package one day !